recipes/ignition-tools/all/test_package/test_package.cpp

```diff
@@ -1,7 +1,5 @@
 #include <iostream>
 
-
-int main(int argc, char **argv)
-{
+int main() {
```

Comment:
Ugh... I looked into what this package actually should provide and this recipe is quite incorrect.

The built library is meant to be a pure runtime library for the `ign` Ruby script packaged by this recipe. CMake and pkg-config targets should not be exported by the recipe, since they are effectively empty. Here's the .pc file installed by the project:
```
# This packages provides no headers or libraries, so those fields are omitted
Name: Ignition tools
Description: A set of tools classes for robot applications
Version: 1.4.0
```

I updated the recipe accordingly.